lib/workers/global/meta.js

```diff
@@ -23,5 +25,22 @@ function setMeta(config) {
       name: gitAuthorParsed.name,
       email: gitAuthorParsed.address,
     };
+  } else {
+    const credentials = hostRules.find(config, {});
+    let gitAuthorFromApi;
+    try {
+      gitAuthorFromApi = await getPlatformApi(config.platform).getAuthor(
+        credentials.token,
+        credentials.endpoint
+      );
+    } catch (err) {
+      logger.debug('Error getting gitAuthor from api');
+    }
+    if (gitAuthorFromApi && gitAuthorFromApi.name && gitAuthorFromApi.email) {
+      global.gitAuthor = {
```

Comment:
Is this compatible with how we handle a manually configured gitAuthor? I think that we only want to use the autodetect one if there is no gitAuthor provided by the user in config.

Reply:
yes, this will only run if `gitAuthor` is absent in config